src/T2DMap.cpp

```diff
@@ -2800,28 +2800,25 @@ void T2DMap::mousePressEvent(QMouseEvent* event)
 
             if (!mMapViewOnly) {
                 if (selectionSize == 0) {
-                    mpCreateRoomAction = new QAction(tr("Create room", "Menu option to create a new room in the mapper"), this);
-                    mpCreateRoomAction->setToolTip(tr("Create a new room here"));
+                    mpCreateRoomAction = new QAction(tr("Create new room here", "Menu option to create a new room in the mapper"), this);
                     connect(mpCreateRoomAction.data(), &QAction::triggered, this, &T2DMap::slot_createRoom);
                     popup->addAction(mpCreateRoomAction);
                 }
 
                 if (selectionSize > 0) {
-                    auto moveRoom = new QAction(tr("Move", "2D Mapper context menu (room) item"), this);
-                    moveRoom->setToolTip(tr("Move room", "2D Mapper context menu (room) item tooltip"));
+                    auto moveRoom = new QAction(tr("Move room(s)", "2D Mapper context menu (room) item", selectionSize), this);
                     connect(moveRoom, &QAction::triggered, this, &T2DMap::slot_moveRoom);
                     popup->addAction(moveRoom);
                 }
 
                 if (selectionSize > 0) {
-                    auto roomExits = new QAction(tr("Exits", "2D Mapper context menu (room) item"), this);
-                    roomExits->setToolTip(tr("Set room exits", "2D Mapper context menu (room) item tooltip"));
+                    auto roomExits = new QAction(tr("Set room exits...", "2D Mapper context menu (room) item"), this);
```

Comment:
This is more than "Setting" room exits.. how about:
```suggestion
                    auto roomExits = new QAction(tr("Adjust room exits/stubs...", "2D Mapper context menu (room) item"), this);
```

Reply:
No, I don't think so,
All actions start with "Set" and not "Adjust" because settings are adjusted there. If changed here, needs to change everywhere.
Stubs are special kinds of exits, so they are included in this description as well.